packages/react-components/src/components/styles/InputBoxComponent.style.ts

```diff
@@ -24,7 +24,7 @@ export const inputBoxStyle = mergeStyles({
   height: '2.25rem',
   lineHeight: '1.5rem',
   '::-webkit-input-placeholder': {
-    fontSize: '0.875rem'
+    fontSize: '1rem' // Prevents screen zooming on iOS
```

Comment:
Can we just remove this CSS and let browser to use the default behaviour instead?

Reply:
this doesn't work.